tests/data/specs/in_array_message.rflx

```diff
@@ -0,0 +1,7 @@
+with Array_Message;
+
+package In_Array_Message is
+
+   for Array_Message::Message_With_SDU use (Payload => Array_Message::Foo);
+
+end In_Array_Message;
```

Comment:
Having a separate package for that seems unnecessary.

Reply:
Removed it in 55b15af